src/openrct2/interface/Widget.h

```diff
@@ -102,14 +102,19 @@ namespace OpenRCT2
 
         int16_t width() const
         {
-            return right - left;
+            return right - left + 1;
         }
 
         int16_t height() const
         {
             return bottom - top;
         }
 
+        int16_t actualHeight() const
+        {
+            return bottom - top + 1;
+        }
```

Comment:
This does not seem to be used yet. To be honest, I'd prefer to avoid introducing an `actualHeight` to the codebase. I see you started with a similar `actualWidth` for the `width` method, though. Do you plan to tackle this in a future PR? If so, perhaps best to introduce the new function at that point, too.